.drone.star

```diff
@@ -127,6 +129,23 @@ config = {
                 "IDP_ACCESS_TOKEN_EXPIRATION": 30,
             },
         },
+        "ocm": {
+            "earlyFail": True,
+            "skip": False,
```

Comment:
@ScharfViktor should this be nightly only thing or run on PRs as well?

Reply:
let's run ocm suite at night and when [full-ci]